lib/FSi/DoctrineExtensions/Uploadable/Mapping/Driver/Yaml.php

```diff
@@ -44,12 +50,17 @@ protected function loadExtendedClassMetadata(ClassMetadataInfo $baseClassMetadat
                             $extendedClassMetadata->getClassName()
                         ));
                     }
+
+                    $keyLength = $this->getValue($uploadable, 'keyLength');
+                    if (!is_null($keyLength)) {
+                        $keyLength = (int) $keyLength;
+                    }
                     $extendedClassMetadata->addUploadableProperty(
                         $field,
                         $this->getValue($uploadable, 'targetField'),
```

Comment:
maybe replace all `getValue` method invokations with a construct like `$uploadable['targetField'] ?? null` ?

Reply:
Next PR, this one is already too big.